augur/export_v2.py

```diff
@@ -103,6 +111,8 @@ def _get_values(key):
                 values_in_tree.add(node_properties.get(key, {}).get("value"))
             elif key in node_properties['traits']:
                 values_in_tree.add(node_properties['traits'][key]['value'])
+        if len(values_in_tree)==1 and None in values_in_tree:
+            return None
```

Comment:
would be more consistent to return an empty set.

Reply:
Agreed -- this is a bug in (my) `values_in_tree` -- it shouldn't contain `None` (but may contain `False` if it's a Boolean variable)